### DISCUSSION TITLE
Scene mangement with builtin pipeline

### DISCUSSION THREAD

**Discussion by mathisloge** [SanderMertens/flecs#1323] [Q&A]
Hi, 

I'm quite new to flecs. 
I was wondering on how to combine the builtin pipeline with the scene_management example from here: https://github.com/SanderMertens/flecs/blob/master/examples/cpp/game_mechanics/scene_management/src/main.cpp

What I've tried is to use the given query from the documentation and add a `.without<MenuScene>` resulting in 

```
    flecs::entity game = ecs.pipeline()
                             .with(flecs::System)
                             .with(flecs::Phase)
                             .cascade(flecs::DependsOn)
                             .without(flecs::Disabled)
                             .up(flecs::DependsOn)
                             .without(flecs::Disabled)
                             .up(flecs::ChildOf)
                             .without<GameScene>()
                             .build();
``` 

However, now the systems tagged with `OnStart` are running constantly.

Is there an easy way to use the builtin pipeline for each scene but with the `    .without<Scene>()` tag?

**Comment by SanderMertens** [comment]
Try adding this to the query:
```cpp
.without(flecs::DependsOn, flecs::OnStart).self().up(flecs::DependsOn)
```
That should prevent `OnStart` systems from getting matched, which is also what the builtin pipeline does.
